app/views/posts/new.html.erb

```diff
@@ -0,0 +1,18 @@
+<h1>Add new Post</h1>
+
+<%= form_with model: [@post], url: user_posts_path(@user), method: :post do |form| %>
```

Comment:
- [ ] Well done :clap: you did the changes given by the previous reviewer :point_up: but still there are some problems in your project when I try to open the `/users/1/posts/new` page it gives me this error: "undefined method `model_name` for nil:NilClass"  which means you model name is not defied so what you will do you just remove the array brackets from your `@post` variable this will solve your error :slightly_smiling_face:  check below :point_down:  my suggestion for more clarification :+1: 

```suggestion
<%= form_with model: @post, url: user_posts_path(@user), method: :post do |form| %>
```

![Screenshot from 2024-01-28 00-03-39](https://github.com/supreeti/blogapp/assets/45226848/179e26fd-103e-4190-bc3a-214f474e4856)